src/rdo.rs

```diff
@@ -78,14 +78,14 @@ impl RDOType {
 }
 
 #[derive(Clone)]
-pub struct RDOOutput {
+pub struct PartitionGroupParameters {
   pub rd_cost: f64,
   pub part_type: PartitionType,
-  pub part_modes: Vec<RDOPartitionOutput>,
+  pub part_modes: Vec<PartitionParameters>,
 }
 
 #[derive(Clone)]
-pub struct RDOPartitionOutput {
+pub struct PartitionParameters {
```

Comment:
If you see the fields in this structure, they are all about the modes of one parittion, that are decided by rdo_mode_decision( ).
So, I don't think the new name 'Parameters' is more implying that meaning than current 'RDO'?
Current one has been not bad, or I need ever want to revise it again, I'd go simply swap two parts there, i,e. PartitionRDOResult, but that can easily mislead us to think it is about results of rdo-based partition search.
Thus, I'd end up with RDOOutput, i.e use already taken name for above struct, which can confuse us! :)

Reply:
It also contains parameters for transform size and type, motion vectors and other data that is not related to the prediction mode.